### PR TITLE
Fix/accessability features

### PR DESCRIPTION
**Pull request by Shawntru**
## Is this a fix or a feature?
Fix
## What is the change?
Adjust font colors, page colors, and HTML tags to increase accessibility 
## What does it fix?
Increases Lighthouse accessibility testing to 100%
## Where should the reviewer start?
Style.scss, index.html
## How should this be tested?
Run lighthouse/Axe testing, desktop testing
## Relevant Ticket(s)
#18  #19 

Screen Shot 2020-10-30 at 7 13 28 PM

@Shawntru 
@adavidson02
@sarah01123

### CODE REVIEW

File: src/css/style.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
 $background-color: #4d4e4f;
 $h1-font-color: #ffffff;
 $body-font-family: 'Open Sans';
-$sidebar-color: #9A9DAB;
+$sidebar-color: lightgrey;
```
Comment (saraho1123) [inline src/css/style.scss:4]: Thanks for increasing the Lighthouse score!!

File: src/index.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (saraho1123) [inline src/index.html:2]: Thanks for remembering this piece!

**Review by saraho1123 — APPROVED** [review]
That is awesome you got the Lighthouse score to 100!

File: src/css/style.scss [hunk 3]
```diff
@@ -173,25 +173,25 @@ h1 {
   flex-direction: column;
   justify-content: flex-start;
   width: 50%;
-  background-color: #5BC8AC;
+  background-color: $hydration-background;
```
Comment (ADavidson02) [inline src/css/style.scss:176]: This is a much better name! Thank you!

**Review by ADavidson02 — APPROVED** [review]
Looks amazing!

**Pull request merged**